build.js

```diff
@@ -14,6 +14,7 @@
 
 'use strict';
 
+process.env.SERVER_ROOT = "https://models.accordproject.org";
```

Comment:
Instead of hardcoding this in build.js can you please set it in package.json where the script is called?